how-to/panel-login.md

```diff
@@ -0,0 +1,67 @@
+---
+id: panel-login
```

Comment:
```suggestion
```
`id` only needs to be explicitly specified if it differs from the filename. See: https://github.com/PlaceOS/docs#id.